frontend/home/home.component.tsx

```diff
@@ -25,16 +25,16 @@ export default function Home(props: HomeProps) {
           title="Add a new client"
           link="add-client"
         />
-        <HomeCard
-          iconUrl={caseNoteUrl}
-          title="Add a case note"
-          link="add-case-note"
-        />
         <HomeCard
           iconUrl={recordVisitUrl}
           title="Add a client interaction"
           link="add-client-interaction"
         />
+        <HomeCard
```

Comment:
Just to clarify, why are we moving this card?

Reply:
Good question -- it's to match the ordering of the features in the sidenav. The homepage and sidenav listed the features in different orders.